### PR TITLE
Aegires Nerf

### PR DESCRIPTION
**Pull request by Zerenta**
Nerfed healing (3% to 1.5% on non-aegires, 0.5% on aegires) because 3% resulted in large groups of aegires healing eachother faster than damage (from endgame, surge tier defenses (aegires is phase tier)) could be dealt, 1 omura (a T5) has large issues trying to kill 2 aegires (T4s) when the healing is 3%
Buffed segments (reload 6 frames to 4 frames, 25 damage to 30 damage)
This is because aegires are overpowered as hell when in groups

If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [x] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [x] I have ensured that my code compiles, if applicable.
- [x] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

**Comment by UnionofSovietSocialistRepublics** [comment]
We demand aegries nerd 🗣️📣🔥🔥🔥

**Comment by Lime712** [comment]
Nerb now! ⛈️🗣️🔊📢📣🔥🧙

**Comment by Gwapowo** [comment]
Nerp yourself, now! 🗣️📣💥

File: core/src/mindustry/entities/abilities/EnergyFieldAbility.java [hunk 1]
```diff
@@ -28,7 +28,7 @@ public class EnergyFieldAbility extends Ability{
     public float x, y;
     public boolean targetGround = true, targetAir = true, hitBuildings = true, hitUnits = true;
     public int maxTargets = 25;
-    public float healPercent = 3f;
+    public float healPercent = 1f;
```
Comment (BalaM314) [inline core/src/mindustry/entities/abilities/EnergyFieldAbility.java:31]: This is not the right way to do this, you are changing the default setting, which will affect mods. Change it in UnitTypes::aegires instead. (add a line inside the init block `healPercent = 1f`)

**Review by BalaM314 — CHANGES_REQUESTED** [review]

**Comment by BalaM314** [comment]
Aegires is absolutely broken in groups and definitely needs a nerf, but 3% to 1% might be a bit much. Maybe consider nerfing healing between aegires? (aegires heals 1.5% on other units, but only 0.5% on other aegires) I think that could work, stops aegires from being unkillable while keeping them useful as a support unit)
We also need to consider campaign, where the unit cap is 24... would probably still be fine

File: core/src/mindustry/entities/abilities/EnergyFieldAbility.java (hunk 1 above)
Comment (Zerenta) [reply core/src/mindustry/entities/abilities/EnergyFieldAbility.java]: Tysm! (For others reading this has been fixed)

**Comment by UnionofSovietSocialistRepublics** [comment]
Halai

**Comment by Zerenta** [comment]
> 



> Aegires is absolutely broken in groups and definitely needs a nerf, but 3% to 1% might be a bit much. Maybe consider nerfing healing between aegires? (aegires heals 1.5% on other units, but only 0.5% on other aegires) I think that could work, stops aegires from being unkillable while keeping them useful as a support unit) We also need to consider campaign, where the unit cap is 24... would prob

**Comment by Zerenta** [comment]
Shit accidental

**Comment by Zerenta** [comment]
> Aegires is absolutely broken in groups and definitely needs a nerf, but 3% to 1% might be a bit much. Maybe consider nerfing healing between aegires? (aegires heals 1.5% on other units, but only 0.5% on other aegires) I think that could work, stops aegires from being unkillable while keeping them useful as a support unit) We also need to consider campaign, where the unit cap is 24... would probably still be fine

Ye a 1.5% heal was considered but with how op 3% was, it felt like 1% was the only way to make it actually reasonable
I also don't know how to do 0.5 to just other aegires

**Comment by Lime712** [comment]
we want nerve!

**Comment by BalaM314** [comment]
> > Aegires is absolutely broken in groups and definitely needs a nerf, but 3% to 1% might be a bit much. Maybe consider nerfing healing between aegires? (aegires heals 1.5% on other units, but only 0.5% on other aegires) I think that could work, stops aegires from being unkillable while keeping them useful as a support unit) We also need to consider campaign, where the unit cap is 24... would probably still be fine
> 
> Ye a 1.5% heal was considered but with how op 3% was, it felt like 1% was the only way to make it actually reasonable I also don't know how to do 0.5 to just other aegires

Will do (ofc it needs to be generic)

**Comment by UnionofSovietSocialistRepublics** [comment]
Nerf aegries to the ground

File: core/src/mindustry/ai/UnitCommand.java [hunk 2]
```diff
@@ -19,6 +19,10 @@ public class UnitCommand{
         drawTarget = true;
         resetTarget = false;
     }},
+    ramCommand = new UnitCommand("ram", "modeAttack", u -> null){{
```
[the remark this answers is not in the document]
Comment (Zerenta) [reply core/src/mindustry/ai/UnitCommand.java]: yeah my bad

**Pull request merged**